domain/structs.go

```diff
@@ -85,10 +86,17 @@ type APIInfo struct {
 
 // RestAPIDetail provides details about an endpoint
 type RestAPIDetail struct {
-	CommandName string         `json:"summary"`
-	JQFilter    string         `json:"jqFilter"`
-	Consumes    []string       `json:"consumes"`
-	Parameters  []RestAPIParam `json:"parameters"`
+	CommandName string          `json:"summary"`
+	JQFilter    string          `json:"jqFilter"`
```

Comment:
You will need the ability to also parse `x-jqFilter` for the changes made in the newer API. I assume you can simply add an element here and do something similar to what you have done for `Definitions` and `Components`.